### DISCUSSION TITLE
Data for "How many prayed today"

### DISCUSSION THREAD

**Discussion by ansleyglenn** [DiscipleTools/disciple-tools-prayer-campaigns#367] [Q&A] [answered]
Hello - I have a quick question I was hoping someone could confirm for me.

On the prayer fuel page, there is a form you can submit that asks "How many prayed today". I see that this gets stored in the dt_reports table with a type of 'fuel' and a subtype of 'ongoing'. I would like to ask / confirm that this is the only place that these values get entered into the DB in that manner. 

In other words, if I have 100 records in the dt_reports table with a type of 'fuel' and a subtype of 'ongoing' for a specific campaign, can I assume that this button was clicked 100 times? Or is there other circumstances that also add this data to the table?

Thanks!

**Comment by corsacca** [comment, marked as answer]
Hey @ansleyglenn,

Yes you are correct. Records in the dt_reports table with a type of 'fuel' and a subtype of 'ongoing' for a specific campaign are only added from a user using the "How many prayed today" form.
